src/PersonObjects/Player/PlayerObjectGeneralMethods.tsx

```diff
@@ -1794,6 +1795,16 @@ export function applyForJob(this: IPlayer, entryPosType: CompanyPosition, sing =
   }
 
   this.jobs[company.name] = pos.name;
+
+  if (this.sleeves !== null) {
+    for (const sleeve of this.sleeves) {
+      if (sleeve.currentTask === SleeveTaskType.Company && sleeve.currentTaskLocation === company.name) {
+        sleeve.finishTask(this);
+        sleeve.workForCompany(this, company.name);
+      }
+    }
+  }
```

Comment:
and this feels out of place, like again if the player is applying to work for X wont this just assume you want 1 of the sleeves to work for this same company X also?

Reply:
I think not, it's only doing anything if one of your sleeves _already_ works in the concerned company. 
Idle sleeve, and sleeve working for another company won't enter the if block.

We should be fine.

But yeah, nonetheless, PlayerObjectGeneralMethod interacting directly with sleeve is awkward.

Reply:
Reverted that change.